app/shared/date-picker/_date-picker.scss

```diff
@@ -24,7 +24,8 @@
     position: absolute;
     z-index: 100;
     border: solid 2px $brand-primary;;
-    padding: 20px;
+    padding: 12px;
+    padding-bottom: 0;
```

Comment:
Now you are just overriding 12px padding from the line above, you should combine these paddings (https://developer.mozilla.org/en-US/docs/Web/CSS/padding)